compat/test/browser/forwardRef.test.js

```diff
@@ -351,4 +352,33 @@ describe('forwardRef', () => {
 		expect(spy).to.be.calledOnce;
 		expect(spy).to.be.calledWithExactly({ foo: 100 });
 	});
+
+	it('stale ref missing with passed useRef', () => {
+		let _ref = null;
+		let _set = null;
+		const Inner = forwardRef((props, ref) => {
+			const _hook = useState(null);
+			_ref = ref;
+			_set = _hook[1];
+			return <div ref={ref} />;
+		});
+
+		const Parent = () => {
+			const parentRef = useRef(null);
+			return <Inner ref={parentRef}>child</Inner>;
+		};
+
+		act(() => {
+			render(<Parent />, scratch);
+		});
+
+		expect(_ref).not.to.be.undefined;
```

Comment:
Could we be more explicit with what we expect in these lines? `not undefined` could let other bugs slip through since its very generic. For example `expect(ref.current).to.be.null` and `expect(ref.current).to.equalNode(scratch.firstChild)` might be better

Reply:
Hey @marvinhagemeister and @andrewiggins, those are great suggestions! To be honest, I think I was trying to make sure that the initial `ref` was still there, but not particularly any shape except not undefined. Functionally I guess it doesn't make sense to expect the initial value, but just wanted to make sure we're gonna have a normal code flow in the future.

I'll add in your suggestions! Priceless!